example/SteeringFile_IDEA_o1_v03.py

```diff
@@ -0,0 +1,651 @@
+## 
+##   Create a custom filter. The dictionary is used to instantiate the filter later on
+##   >>> SIM.filter.filters['edep3kev'] = dict(name="EnergyDepositMinimumCut/3keV", parameter={"Cut": 3.0*keV} )
+## 
+##    
+################################################################################
+
+## 
+##     default filter for calorimeter sensitive detectors;
+##     this is applied if no other filter is used for a calorimeter
+##     
+SIM.filter.calo = "edep0"
+
+##  list of filter objects: map between name and parameter dictionary 
+SIM.filter.filters = {'geantino': {'name': 'GeantinoRejectFilter/GeantinoRejector', 'parameter': {}}, 'edep1kev': {'name': 'EnergyDepositMinimumCut', 'parameter': {'Cut': 0.001}}, 'edep0': {'name': 'EnergyDepositMinimumCut/Cut0', 'parameter': {'Cut': 0.0}}}
+
+##  a map between patterns and filter objects, using patterns to attach filters to sensitive detector 
+SIM.filter.mapDetFilter = {}
+
+##  default filter for tracking sensitive detectors; this is applied if no other filter is used for a tracker
+SIM.filter.tracker = "edep1kev"
+
+
+################################################################################
+## Configuration for the Detector Construction. 
+################################################################################
+SIM.geometry.dumpGDML = ""
+SIM.geometry.dumpHierarchy = 0
+
+## Print Debug information about Elements
+SIM.geometry.enableDebugElements = False
+
+## Print Debug information about Materials
+SIM.geometry.enableDebugMaterials = False
+
+## Print Debug information about Placements
+SIM.geometry.enableDebugPlacements = False
+
+## Print Debug information about Reflections
+SIM.geometry.enableDebugReflections = False
+
+## Print Debug information about Regions
+SIM.geometry.enableDebugRegions = False
+
+## Print Debug information about Shapes
+SIM.geometry.enableDebugShapes = False
+
+## Print Debug information about Surfaces
+SIM.geometry.enableDebugSurfaces = False
+
+## Print Debug information about Volumes
+SIM.geometry.enableDebugVolumes = False
+
+## Print information about placements
+SIM.geometry.enablePrintPlacements = False
+
+## Print information about Sensitives
+SIM.geometry.enablePrintSensitives = False
+
+
+################################################################################
+## Configuration for the GuineaPig InputFiles 
+################################################################################
+
+## Set the number of pair particles to simulate per event.
+##     Only used if inputFile ends with ".pairs"
+##     If "-1" all particles will be simulated in a single event
+##     
+SIM.guineapig.particlesPerEvent = "-1"
+
+
+################################################################################
+## Configuration for the DDG4 ParticleGun 
+################################################################################
+
+##  direction of the particle gun, 3 vector 
+SIM.gun.direction = (0, 0, 1)
```

Comment:
The test would now shoot particles in the beampipe and thus not cross any detector, could you change this?

Reply:
@BrieucF interesting, the steering file setup other than the SD and the file I/O part should be identical to other `CTest` setups since he used `--dumpSteeringFile` to configure the steering file. Could this mean that the same thing is happening for other `CTest` as well?

Reply:
Yes, it seems so

Reply:
I guess it matters less since most of the other subdetector uses "central" sensitive actions. Unless the sensitive action are being called anyway at some point, I would make sure we actually have some activity in the dual-readout so that this is also being 'tested'

Reply:
I updated direction to (1., 1., 1.)